qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -532,3 +533,11 @@ def _parse_approximation_degree(approximation_degree):
     if approximation_degree < 0.0 or approximation_degree > 1.0:
         raise TranspilerError("Approximation degree must be in [0.0, 1.0]")
     return approximation_degree
+
+
+def _parse_seed_transpiler(seed_transpiler):
+    if seed_transpiler is None:
+        return None
+    if not type(seed_transpiler) == int or seed_transpiler < 0:
```

Comment:
Lint is complaining about the type-checking method, see https://dev.azure.com/qiskit-ci/qiskit/_build/results?buildId=59920&view=logs&j=a15d9aca-d75c-55b1-7f76-889cfe884cf0&t=154f6391-7dd7-5733-2230-2f10f0798b76&l=618

I think Lint would prefer sth like:

```suggestion
    if not isinstance(seed_transpiler, int) or seed_transpiler < 0:
```